frontend/src/lib/api.ts

```diff
@@ -393,8 +397,8 @@ export function usePresets(platform: string): Preset[] {
     };
 
     const url = typeof platform === "string" ? "/preset" : null;
-    const { data } = useSWR([url, platform], getByPlatform, {
-        refreshInterval: 0,
+    const { data } = useSWRImmutable([url, platform], getByPlatform, {
+        refreshInterval: 1000 * 60 * 60, // 1 hour
```

Comment:
this one and usePreset I think should be reduced a lot if not reverted to be a normal useSWR thing